lighthouse-core/audits/accessibility/axe-audit.js

```diff
@@ -53,6 +53,8 @@ class AxeAudit extends Audit {
           selector: Array.isArray(node.target) ? node.target.join(' ') : '',
           path: node.path,
           snippet: node.html || node.snippet,
+          textContent: node.textContent,
```

Comment:
I added the text content to the rendered elements to make them easier to read. If we only show the element screenshots on hover/tap then I think it's worth showing. What do you think?

Do we need the property for that? Yes because the HTML may be truncated, so we can't extract the text from it. But we can limit it to 80 characters or so.

![Screenshot 2019-05-15 at 09 48 31](https://user-images.githubusercontent.com/1303660/57762067-ab54bb00-76f6-11e9-8e81-64f79db8d0d7.png)

Reply:
Maybe we should call it `text` and we can also count `alt` or `aria-label` attributes if there's no textContent

Reply:
Hm, might be difficult if we're specifically flagging elements that are using `alt`/`aria-label` incorrectly 😆 

how about we stick to textContent for now but limit to 80 characters as you suggest? 👍

Reply:
I think using `alt/aria-label` is quite helpful, e.g. for buttons that just consist of an image.

Created a PR here: https://github.com/GoogleChrome/lighthouse/pull/8961

Reply:
Oh if you've already done it then more power too ya! 👍 ❤️